### PR TITLE
Add a line break after warning 👼

### LINKED ISSUE #234: No line break after message about configs in stack file
**Description**

The Docker 17.06 client generates a warning about the `configs` option in stack services when communicating with older engines (i.e., less than API version 1.30). However, the warning appears to be missing a line break, resulting in further messages being outputted at the end of the warning.

**Steps to reproduce the issue:**

In this example, I am using the CLI bundled with Docker for Mac 17.06.0-ce-rc5-mac16 to remove a stack deployed on a Docker CE 17.05.0 swarm.

1. Remove an already deployed stack.
```
docker stack rm exporters
```

**Describe the results you received:**

The stack is removed successfully. But, there is a missing newline between the warning about ignoring "configs" and the message about removing the first service. Note, you might need to scroll the quoted block to the right to see it.

```
(rmsdev) →  ~ docker stack rm exporters
WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is 1.29)Removing service exporters_cadvisor
Removing service exporters_node-exporter
Removing network exporters_default
```

**Describe the results you expected:**

There should be a line break between the configs warning and the line about removing service exporters_cadvisor. Something like the following

```
(rmsdev) →  ~ docker stack rm exporters
WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is 1.29)
Removing service exporters_cadvisor
Removing service exporters_node-exporter
Removing network exporters_default
```

**Additional information you deem important (e.g. issue happens only occasionally):**

The warning became visible after docker/cli#162 fixed handling of configs with old daemons.

**Output of `docker version`:**

```
Client:
 Version:      17.06.0-ce-rc5
 API version:  1.29 (downgraded from 1.30)
 Go version:   go1.8.3
 Git commit:   b7e4173
 Built:        Tue Jun 20 07:14:13 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:10:29 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Output of `docker info`:**

```
Containers: 68
 Running: 21
 Paused: 0
 Stopped: 47
Images: 693
Server Version: 17.05.0-ce
Storage Driver: overlay
 Backing Filesystem: xfs
 Supports d_type: true
Logging Driver: gelf
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
 Log: 
Swarm: active
 NodeID: 5i7ghc9agzge2hqfrmrgcucvn
 Is Manager: true
 ClusterID: m0gz05zqgmhx67noqdvko2npr
 Managers: 3
 Nodes: 10
 Orchestration:
  Task History Retention Limit: 5
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
  Force Rotate: 0
 Root Rotation In Progress: false
 Node Address: 10.41.149.137
 Manager Addresses:
  10.41.149.137:2377
  10.41.149.138:2377
  10.41.149.139:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 9048e5e50717ea4497b757314bad98ea3763c145
runc version: 9c2d8d184e5da67c95d601382adf14862e4f2228
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 3.10.0-514.21.1.el7.x86_64
Operating System: Red Hat Enterprise Linux Server 7.3 (Maipo)
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 15.51GiB
Name: itrmsdev01.ucalgary.ca
ID: MYIJ:6PFS:CZR5:QGU6:2ZBA:S3T7:3ER5:3QRM:5QYB:ZM22:SH6N:S2JP
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Client: Docker CE for Mac 17.06.0-rc5
Server: Docker CE 17.05.0 on RHEL 7.3

### PR DESCRIPTION
**Pull request by vdemeester**
Fix #234 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: cli/command/stack/remove.go [hunk 1]
```diff
@@ -63,7 +63,8 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 			return err
 		}
 		if versions.LessThan(version.APIVersion, "1.30") {
-			fmt.Fprintf(dockerCli.Err(), `WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is %s)`, version.APIVersion)
+			fmt.Fprintf(dockerCli.Err(), `WARNING: ignoring "configs" (requires API version 1.30, but the Docker daemon API version is %s)
```
Comment (dnephin) [inline cli/command/stack/remove.go]: Where's the `\n`... Oh!

Maybe ` + "\n"` or just just use `" .. \" ... \" " `?

Comment (vdemeester) [inline cli/command/stack/remove.go]: yeah I'll update

Comment (thaJeztah) [inline cli/command/stack/remove.go]: Ah, yes, I'd prefer `\n` or something as well :)

Comment (vdemeester) [inline cli/command/stack/remove.go]: done 😉

**Comment by thaJeztah** [comment]
@vdemeester I see you attached this to the 17.06 milestone; is this important enough? (would probably be 17.06.1 if it is)

**Comment by vdemeester** [comment]
@thaJeztah yeah I was thinking about 17.06.1 😉 (but there is no milestone for it yet 😛)

**Comment by thaJeztah** [comment]
there - fixed it :)

**Pull request merged**